FastAPI Assignment 2/app/utils/exceptions.py

```diff
@@ -0,0 +1,67 @@
+from http import HTTPStatus
+
+
+class RaiseError(Exception):
+    def get_message(self):
+        pass
+
+    def set_status(self):
+        pass
+
+
+class InvalidIDError(Exception):
+    def __init__(self, invalid_id: int, item_type: str):
+        self.invalid_id = invalid_id
+        self.item_type = item_type
+        self.status = HTTPStatus.BAD_REQUEST
+        super().__init__()
+
+    def get_message(self):
```

Comment:
Instead of implementing two separate methods you can override the `__str__` method and provide all the info in that.